### ISSUE TITLE
amp-video-docking: Cannot set video z-index, video is underneath content

### ISSUE THREAD

**Issue by a-haan** [ampproject/amphtml#26121]
I have implemented amp-video-docking on my site, it has issues but I decided to keep it live to show you the issue.  https://ahaan.co.uk/article/top_stories/iptvx-for-apple-tv

Whilst the video is docked, scroll down and the video will disappear behind the content.  The shadow will stay above content.

I currently can't find a way to change the z-index for the video myself.

**Comment by caroqliu** [comment]
Thanks for posting this issue. I was able to reproduce it with the steps you included. I see that the video disappears behind some of the content but not all. Is it possible to modify the z-index on the content it is disappearing behind? Here is a [list of `z-index` by selector](https://github.com/ampproject/amphtml/blob/master/css/Z_INDEX.md) if that helps.

Ads also appear to be sandwiching the docked video (over the video and under the shadow and controls)--we should iron out an approach here. /cc @calebcordry

**Comment by calebcordry** [comment]
Looks like the `z-index` on ads side will be changing soon see https://github.com/ampproject/amphtml/issues/24653#issuecomment-566777888

cc/ @glevitzky

**Comment by a-haan** [comment]
I've tried changing the z-index for the elements that are overlapping with no success.

**Comment by a-haan** [comment]
I had unknowingly set the z-index of amp-youtube to 1 - issue is now fixed after removing this css.
